jbpm-human-task/jbpm-human-task-audit/src/main/java/org/jbpm/services/task/audit/JPATaskLifeCycleEventListener.java

```diff
@@ -920,5 +945,16 @@ protected String getActualOwner(Task ti) {
         
         return userId;
     }
+    
+    private void createView (TaskEvent event, TaskEventType type) {
+        createView (event, type, null);
+    }
+    
+    private void createView (TaskEvent event, TaskEventType type, AssignmentType assignType) {
```

Comment:
Minor comment: perhaps a name like `emitTaskOperationEvent()` or `generateTaskOperationEvent()`  or `sendTaskOperationView()` or similar would clarify the purpose of this method

Reply:
Actually since this method is always called after `persist()`, perhaps `persist()` can be included as part of the method and extend it's responsibility (with a new method name too)

Reply:
I did not want to change persist on purpose, the creation of the view is responsabiilty of the event listener, not the task context.
Regarding the name, I will change to createTaskOperationView (Im not fan of long names for methods, but it is ok)